app/components/project.tsx

```diff
@@ -0,0 +1,35 @@
+interface ProjectProps {
+    image?: string;
```

Comment:
since we allow the props to be undefined, we should set some defaults within the component